### PR TITLE
(fix): style fixes  + blog banner cookie

### PR DESCRIPTION
**Pull request by anamontiaga**
### Overview

- [x] Save blog banner on local storage 
- [x] Add blog banner to mobile
- [x] Add hover states on buttons and menu options
- [x] Review press states on menu icons 
- [x] Review icon sizes on sidebar
- [x]  Add background to Species Location widget legend.
- [x]  Increase desktop sidebar width to join widgets area.
- [x]  Add hover state on category dialogs options.
- [x] Add hover state on location list

### CODE REVIEW

File: src/components/highlighted-places/index.tsx [hunk 1]
```diff
@@ -25,7 +25,7 @@ const HighlightedPlaces = () => {
   const isHighlightedPlace = Object.values(HIGHLIGHTED_PLACES).includes(location_id);
 
   return (
-    <div className="flex justify-between space-x-6">
+    <div className="flex justify-between space-x-6 px-10">
```
Comment (mluena) [inline src/components/highlighted-places/index.tsx:28]: do we need both space-x and px-10?

File: src/containers/widgets/index.tsx [hunk 2]
```diff
@@ -48,7 +69,10 @@ const WidgetsContainer: React.FC = () => {
           {widgetsCollapsedChecker ? 'Expand all widgets' : 'Collapse all widgets'}
         </button>
       )}
-      <Media greaterThanOrEqual="md">{blogBanner && <Blog setBlogBanner={setBlogBanner} />}</Media>
+
+      <Media greaterThanOrEqual="md">
+        {isBlogActive && <Blog closeBlogBanner={closeBlogBanner} />}
+      </Media>
```
Comment (mluena) [inline src/containers/widgets/index.tsx:75]: Why not on mobile?

File: src/components/highlighted-places/index.tsx (hunk 1 above)
Comment (anamontiaga) [reply src/components/highlighted-places/index.tsx:28]: Yes, px-10 is for padding all around de div and space-x is for space between two elements inside the div

File: src/containers/widgets/index.tsx (hunk 2 above)
Comment (anamontiaga) [reply src/containers/widgets/index.tsx:75]: Added

**Comment by andresgnlez** [comment]
image
 Worldwide icon is not visible on hover.

**Comment by andresgnlez** [comment]
image

- Location search keeps hovering when the dialog is closed.
- Drawing tool icon is not hovered when it is selected.

**Comment by anamontiaga** [comment]
> image image
> 
> * Worldwide icon is not visible on hover.
> * Location search icon keeps hovering when the dialog is closed.
> * Drawing tool icon does not hover when it is selected.

Thnks! fixed except worldwide icon problem that i am not able to reproduce

**Comment by andresgnlez** [comment]
image

Let's double-check the hover styles of the submenu with design.

File: src/containers/locations-list/index.tsx [hunk 3]
```diff
@@ -77,15 +77,15 @@ const LocationsList = ({ onSelectLocation }: { onSelectLocation?: () => void })
     return (
       <CellMeasurer key={key} parent={parent} cache={cache} columnIndex={0} rowIndex={index}>
         {({ registerChild }) => (
-          <div style={style} ref={registerChild}>
+          <div style={style} ref={registerChild} className="no-scrollbar md:px-6">
```
Comment (mluena) [inline src/containers/locations-list/index.tsx:80]: Screenshot 2023-06-14 at 08 35 54
please, review paddings

https://www.figma.com/file/GJCx8yGfx6JERPuxlpG3nK/Mangrove-Watch-%5BInternal%5D?type=design&node-id=0-358&t=e5OeC5PBmOs7zSqx-0

Comment (anamontiaga) [reply src/containers/locations-list/index.tsx:80]: i am not able to identify the bug. 
Screenshot 2023-06-14 at 09 50 40

**Comment by andresgnlez** [comment]
image

The borders are probably inheriting from defaults. Can we change them? I think we have some styles here: https://www.figma.com/file/GJCx8yGfx6JERPuxlpG3nK/Mangrove-Watch-%5BInternal%5D?type=design&node-id=834-4938&t=NCLm3Xos7bse8aXf-0 (I would try with the light green)

File: src/containers/locations-list/index.tsx (hunk 3 above)
Comment (andresgnlez) [reply src/containers/locations-list/index.tsx:80]: Me neither.

**Pull request merged**